contentScript.js

```diff
@@ -0,0 +1,149 @@
+(() => {
+
+	let ytLeftControls, ytPlayer;
+	let currentVid = null;  // id of the currently playing video
+	let lastVid = null;   // id of the last handled video
+	let BmsList = [];  // list of the video bookmarks
+    
+		/* Fetch all bookmarks for current video */
+		const fetchBookmarks = () => {
+			return new Promise(resolve => {
+				// get all bookmarks for current video and create bookmark elements
+				chrome.storage.sync.get([ currentVid ], (obj) => {
+					resolve(obj[currentVid] ? JSON.parse(obj[currentVid]) : [])
+				});
+			});
+		}
+	
+		/* Clear out all previous bookmarks */
+		const delOldBms = () => {
+			// remove bookmarks from previous video
+			const oldBms = document.getElementsByClassName('bookmark');
+			while(oldBms[0]) { oldBms[0].parentNode.removeChild(oldBms[0]);}
+		}
+	
+		/* get current video's duration */
+		const getVideoDuration = () => {
+			if (ytPlayer) { return parseInt(ytPlayer.duration);}
+			return null;
+		}
+
+			/* Handle request to add new bookmark */
+			const addNewBmEventHandler = () => {
+	
+		let currentTime = null;
+		if (ytPlayer) {
+		 currentTime = parseInt(ytPlayer.currentTime);
+		}
+
+		if (currentTime) {
+
+			const newBm = { time: currentTime, desc: 'Bookmark at ' + getTime(currentTime) };
```

Comment:
when using objects I tend to new line for each new property like this.

`
const newBm = { 
  time: currentTime, 
  desc: 'Bookmark at ' + getTime(currentTime) 
};
`

Reply:
lol it didnt add newlines in my code snippet but i think you get the point that we should add this to a new line

Reply:
yup, adding this in my latest pr